yt_dlp/extractor/rai.py

```diff
@@ -548,7 +548,7 @@ class RaiIE(RaiBaseIE):
     _TESTS = [{
         # var uniquename = "ContentItem-..."
         # data-id="ContentItem-..."
-        'url': 'http://www.raisport.rai.it/dl/raiSport/media/rassegna-stampa-04a9f4bd-b563-40cf-82a6-aad3529cb4a9.html',
+        'url': 'https://www.raisport.rai.it/dl/raiSport/media/rassegna-stampa-04a9f4bd-b563-40cf-82a6-aad3529cb4a9.html',
```

Comment:
Shouldn't matter,right?

Reply:
yup, because this test is skipped is unnecessary.

I just changed all `http` to `https` in the code while I was there.